ddspipe_core/src/cpp/logging/DdsLogConsumer.cpp

```diff
@@ -42,13 +42,6 @@ DdsLogConsumer::DdsLogConsumer(
     fastdds::dds::DomainParticipantQos pqos;
     pqos.name("DdsLogConsumerParticipant_" + std::to_string(configuration->publish.domain));
```

Comment:
That was previously implemented to send `TypeInformation` in case of publishing the types. Now, this functionality is handled internally by the `TypeLookupService` in Fast DDS.